echopype/convert/set_groups_ek60.py

```diff
@@ -146,7 +146,7 @@ def set_env(self) -> xr.Dataset:
             ds_env.append(ds_tmp)
 
         # Merge data from all channels
-        ds = xr.merge(ds_env)
+        ds = xr.merge(ds_env, compat="no_conflicts", join="outer")
```

Comment:
it seems that `compat="no_conflicts"` and `join="outer"` are the default values at least in the recent xarray versions. What did you run into that motivated you to add them here explicitly?